expyfun/visual/_visual.py

```diff
@@ -883,9 +881,9 @@ def update_bar(self, percent):
         percent: float
             The percentage of the bar to be filled. Must be between 0 and 1.
         """
-        if percent > 1 or percent < 0:
-            raise ValueError('percent must be a float between 0 and 1')
-        self._pos_bar[2] = percent * self._width
+        if percent > 100 or percent < 0:
+            raise ValueError('percent must be a float between 0 and 100')
+        self._pos_bar[2] = percent * self._width / 100
```

Comment:
the divisor should probably be `100.` to be safe...  I'm not positive but it looks possible that both `percent` and `self._width` could be `int`s, which would make this integer division under py2